src/App.js

```diff
@@ -1,13 +1,18 @@
 import "./App.css";
 import { Header, Footer } from "components";
-import {AllRoutes} from "./AllRoutes";
-
+import { AllRoutes } from "./AllRoutes";
+import { useAuth } from "contexts";
+import { Toast } from "components";
```

Comment:
you can access in toast line number 2 .
I think this line is no need

Reply:
Thanks for pointing it out, I have fixed it in the latest commit